src/wifi/model/dmg-wifi-mac.h

```diff
@@ -765,6 +777,16 @@ class DmgWifiMac : public RegularWifiMac
   /* Access Period Allocations */
   AllocationFieldList m_allocationList;         //!< List of access period allocations in DTI.
 
+  typedef struct 
+    {
+      AllocationID id;
+      uint16_t dstAid;
+    } AllocatedDataStruct;
+
+  typedef std::vector<AllocatedDataStruct> AllocatedRequestsVec;
+  AllocatedRequestsVec m_allocatedRequests;     //!< Vector of requested allocations granted to this STA
+  bool m_accessInCbap;                          //!< Flag to indicate whether a Station with allocated SP/CBAP is allowed to compete in a CBAP.
```

Comment:
keep a consistent naming style
```suggestion
  bool m_accessInCbap;                          //!< Flag to indicate whether a STA with allocated SP/CBAP is allowed to compete in a CBAP.
```